src/test/codegen/instrument-mcount.rs

```diff
@@ -0,0 +1,7 @@
+// ignore-tidy-linelength
+// compile-flags: -Z instrument-mcount
+
+#![crate_type = "lib"]
+
+// CHECK: attributes #{{.*}} "instrument-function-entry-inlined"="{{_*}}mcount" "no-frame-pointer-elim"="true"
```

Comment:
This test will fail on e.g. macOS, won’t it, as thesymbol there is `\01mcount`.

Reply:
I came up with `{{.*}}mcount{{.*}}`, is this correct?

Reply:
Probably, we can try that with bors :)